src/clib/pio_msg.c

```diff
@@ -131,37 +131,21 @@ int create_file_handler(iosystem_desc_t *ios)
     if ((mpierr = MPI_Bcast(&len, 1, MPI_INT, 0, ios->intercomm)))
         return check_mpi2(ios, NULL, mpierr, __FILE__, __LINE__);
     LOG((1, "create_file_handler got parameter len = %d\n", len));
```

Comment:
This LOG line is in existing code. I will create another issue on cleaning up pio_msg.c